applications/openshift/scheduler/scheduler_no_bind_address/tests/ocp4/4.14.yml

```diff
@@ -0,0 +1,3 @@
+---
+default_result: NOT-APPLICABLE
```

Comment:
I don't think these are used since the rule isn't included in any profiles (just the default one).